src/main.rs

```diff
@@ -371,32 +381,54 @@ fn filter_commits<'repo>(config: &Config, commits: Vec<Commit<'repo>>) -> Vec<Co
 struct CommitHours {
     email: Option<String>,
     author_name: Option<String>,
+    breakdown: HashMap<String, Duration>,
     duration: Duration,
     commit_count: usize
 }
 
-fn estimate_author_time(mut commits: Vec<Commit>, email: Option<String>, max_commit_diff: &Duration, first_commit_addition: &Duration) -> CommitHours {
+fn estimate_author_time(mut commits: Vec<Commit>, email: Option<String>, max_commit_diff: &Duration, first_commit_addition: &Duration, display_breakdown: &bool) -> CommitHours {
```

Comment:
Do we use the `display_breakdown` argument here?

Reply:
We don't! Thanks for noticing. Updating!